power.mss

```diff
@@ -56,8 +56,14 @@
     marker-fill: #928f8f;
     marker-width: 3;
   }
-  [power = 'portal'][zoom >= 16] {
-    marker-file: url('symbols/man_made/power_portal_node.svg');
-    marker-width: 3;
+  [power = 'portal'] {
+    [zoom >= 15] {
```

Comment:
Here `power=portal` nodes are rendered from z15, but ways are only rendered from z16. Since `power=minor_line` and `power=pole` features are only shown from z16 and there can be `power=portal` features attached to minor lines, it would be best to change this line back to z16.

Reply:
Agreed this is awkward
Would you agree to display portals (ways/nodes) on zoom 15 as they are more massive than poles?

Towers can also support minor_lines, but they are displayed starting at zoom 14.

Reply:
Size is not as important as usefulness for a general map user. At z15 we are showing a whole town or suburb on one screen: up to a 10 by 5 kilometer area near the equator. At this zoom level (and z14), power towers + `power=line` are still useful orientation landmarks in rural areas. But power portals seem to be exclusively located inside of power plants and substations, and are not found by themselves in rural areas with few other features around. Power poles are rendered as early as z16 not because they are large, but because in rural areas the `power=minor_line` can be useful for orientation, like a hedge or fence (also first rendered at z16), and we need to show the poles to make the minor_line legible. So z16 is best for power portals as well.